assets/state-mig-manager/0600_daemonset.yaml

```diff
@@ -88,10 +90,14 @@ spec:
       - name: mig-parted-config
         configMap:
           name: "FILLED_BY_OPERATOR"
-      - name: run-nvidia
+      - name: run-nvidia-validations
         hostPath:
-          path: "/run/nvidia"
-          type: Directory
+          path: "/run/nvidia/validations"
+          type: DirectoryOrCreate
+      - name: driver-root
+        hostPath:
+          path: "/run/nvidia/driver"
+          type: DirectoryOrCreate
```

Comment:
Out of scope: This is begining to look like the kind of thing we can add to each component in code. Not sure what the best practice is in this case.